cmd/gofish/list.go

```diff
@@ -39,10 +41,28 @@ func newListCmd() *cobra.Command {
 	return cmd
 }
 
-func findFood() []string {
+// When a request comes in for a single food only load that one
+func compare(X []os.FileInfo, Y []string) []os.FileInfo {
+	var ret []os.FileInfo
+
+	for _, x := range X {
+		for _, y := range Y {
+			if x.Name() == y {
```

Comment:
This file check only checks the file name, but doesn't check the file type.

For example, I could write files into the barrel and `compare` would accept that as an acceptable food name.